custom_components/easee/services.yaml

```diff
@@ -133,7 +133,7 @@ set_charger_circuit_dynamic_current:
       example: 10
 
 set_charger_circuit_max_current:
-  description: Set circuit max current
+  description: Set circuit max current OBS! Should not be used by automation (may wear out flash memory if set too frequently).
```

Comment:
Change OBS! to N.B.

Reply:
See your point, I went with "Caution:" which perhaps is cleaner. Agree?

Reply:
Agree